### ISSUE TITLE
Error on line 2572:53: TypeError: undefined is not an object (evaluating 'vehicleData.capabilities.includes')

### ISSUE THREAD

**Issue by kayyovx** [tonesto7/fordpass-scriptable#60]
Received the error: Error on line 2572:53: TypeError:
undefined is not an object (evaluating
'vehicleData.capabilities.includes')

—Please see attachment. 
![53E9E0AC-342E-4B2B-ABD9-17285AD93181](https://user-images.githubusercontent.com/96359672/153102336-99e71436-5e83-476a-8be0-9f1edf70ff88.png)
![FA15DDEE-D3E7-4C1E-9A82-8AADF8BF889A](https://user-images.githubusercontent.com/96359672/153102343-e3a01bf7-20a4-4b32-b0d7-174020791c76.jpeg)

**Comment by alockbox** [comment]
Same thing for me suddenly.

**Comment by chuckreynolds** [comment]
yuuup. +1

**Comment by kayyovx** [comment]
Seems to have resolved apparently; anyone else working ok as well?

**Comment by chuckreynolds** [comment]
yeah seems to be loading now; force refresh works. Assuming something on the Ford app side of things stopped returning an object but I'd assume the Undefined error could be mitigated by some conditional statement(s)
